tools/src/component-generator.ts

```diff
@@ -49,7 +51,7 @@ import * as VueType from "vue";
 const Vue = VueType.default || VueType;
 import <#= it.widgetName #> from "devextreme/<#= it.dxExportPath #>";
 import { VueConstructor } from "vue";
-import BaseComponent from "<#= it.baseComponentPath #>";
+import { <#= it.isExtension ? 'DxExtensionComponent' : 'DxComponent' #> as BaseComponent } from "<#= it.baseComponentPath #>";
```

Comment:
Better move the base component name to templates args.
`import { <#= it.baseComponentName #> as BaseComponent } from "<#= it.baseComponentPath #>"`